src/config/config.js

```diff
@@ -8,10 +8,7 @@ let localePath =  basePath + 'resources/i18n/{{lng}}.json';
 const appMode = typeof(APP_MODE) != "undefined" ? APP_MODE : undefined;
 const nodeEnv = typeof(process.env.NODE_ENV) != "undefined" ? process.env.NODE_ENV : undefined;
 
-
-if (!nodeEnv || nodeEnv.indexOf('development') > -1) {
-  localePath =  basePath + 'resources/i18n/{{lng}}.json';
-} else if (nodeEnv.indexOf('production') > -1) {
+if (nodeEnv.indexOf('production') > -1) {
```

Comment:
nodeEnv may still be undefined here, so this can fail

Reply:
Fixed it in a force push.